src/CustomMissionLog.as

```diff
@@ -59,6 +61,9 @@ class CustomMissionLog
 		isDrag = false;
 		isResize = false;
 		curButtonX = 5;
+		
+		//Check if we are in swl or tsw
+		isSWL = _global.Enums.MainQuestType["e_AreaMission"] != undefined;

```

Comment:
For uniformness sake please replace all `_global.Enums.MainQuestType["e_AreaMission"]` -> `_global.Enums.MainQuestType.e_AreaMission`. I think it should quietly resolve into none/undefined in TSW.

Reply:
I originally wrote those this way because I didn't know how it would resolve and didn't had access to TSW at the time. After a quick test, it seems like you're right. Gonna fix that ASAP.